File: Python-MySQL - Without Docker/app/files/models/mission_crew.py
from files import db


class MissionCrew(db.Model):
    __tablename__ = 'mission_crew'
    user_id = db.Column(
        db.Integer, db.ForeignKey('users.id'), primary_key=True)
    mission_id = db.Column(db.Integer, db.ForeignKey(
        'missions.id'), primary_key=True)

    def __repr__(self):

        return f"{{\"user_id\": \"{self.user_id}\", \"mission_id\": \"{self.mission_id}\"}}"

File: Node-MySQL/app/models/MissionCrew.js
'use strict';
const {
    Model
} = require('sequelize');
module.exports = (sequelize, DataTypes) => {
    class MissionCrew extends Model {
        /**
         * Helper method for defining associations.
         * This method is not a part of Sequelize lifecycle.
         * The `models/index` file will call this method automatically.
         */
        static associate(models) {
            // define association here
        }
    }
    MissionCrew.init({
        userId: {
            type: DataTypes.INTEGER,
            allowNull: false,
            references: {
                model: 'Users',
                key: 'id'
            }
        },
        missionId: {
            type: DataTypes.INTEGER,
            allowNull: false,
            references: {
                model: 'Missions',
                key: 'id'
            }
        }
    },
        {
            sequelize,
            modelName: 'MissionCrew',
            tableName: 'MissionCrew',
            timestamps: false
        });

    MissionCrew.associate = function (models) {
        MissionCrew.belongsTo(models.Users, { foreignKey: 'userId' })
        MissionCrew.belongsTo(models.Missions, { foreignKey: 'missionId' })
    };

    return MissionCrew;
};